engine/installation/linux/debian.md

```diff
@@ -287,13 +312,14 @@ installing Docker.
     Use this command to install the latest version of Docker:
 
     ```bash
-    $ sudo apt-get install docker
+    $ sudo apt-get install docker-engine
```

Comment:
Hi @mstanleyjones I'll verify whether it's docker-ce or docker-engine. I just saw all your notes, and like the added clarity.